380/0/ruby_koans/about_scoring_project.rb

```diff
@@ -0,0 +1,77 @@
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+
+# :reek:UtilityFunction
+def score(dice)
+  score = 0
+  return score if dice.equal?([])
+  frequency = times_in(dice)
+  points = claculated_from(frequency)
```

Comment:
do you really need this variable?
Just leave `claculated_from(frequency)` as return value